common/views/components/StackingTable/StackingTable.tsx

```diff
@@ -153,7 +149,7 @@ const StackingTable: FunctionComponent<Props> = ({
       </thead>
       <tbody>
         {bodyRows.map((row, index) => (
-          <StyledTr plain={plain} key={index}>
+          <StyledTr key={index}>
```

Comment:
sorry this change is unrelated and pushed by mistake but doesn't change anything